<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Обработчик документов</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>База данных документов</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 18px; font-weight: bold; margin: 10px;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="inputGroup">
      <property name="title">
       <string>Ввод данных</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="inputLabel">
         <property name="text">
          <string>Введите текст для обработки:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="originalTextEdit">
         <property name="placeholderText">
          <string>Введите текст здесь...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="processedTextLabel">
         <property name="text">
          <string>Обработанный текст появится здесь</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #f0f0f0; padding: 5px; border: 1px solid #ccc;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout">
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>Добавить в БД</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { background-color: #4CAF50; color: white; padding: 8px; }</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="text">
            <string>Удалить выбранное</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { background-color: #f44336; color: white; padding: 8px; }</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearButton">
           <property name="text">
            <string>Очистить БД</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { background-color: #ff9800; color: white; padding: 8px; }</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="databaseGroup">
      <property name="title">
       <string>База данных</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="statusLabel">
         <property name="text">
          <string>Всего записей: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
